src/background/devtools/external.ts

```diff
@@ -89,7 +89,9 @@ export async function callBackground(
     meta: { nonce, tabId: browser.devtools.inspectedWindow.tabId },
   };
 
-  if (!port.onMessage.hasListeners()) {
+  // Firefox does not support the API yet
+  // https://github.com/pixiebrix/pixiebrix-extension/issues/679
```

Comment:
Do you want full URLs or is the local issue number enough?

```suggestion
  // Firefox does not support the API yet #679
```

Reply:
Full URL is more convenient for me right now, as I don't have issue patterns set up my IDE. But either way works, as GitHub automatically links the issue in the code

Reply:
> GitHub automatically links the issue in the code

\* Refined GitHub 😃